static/app/components/notificationActions/notificationActionManager.tsx

```diff
@@ -170,15 +186,18 @@ function NotificationActionManager({
       ) {
         return;
       }
-
+      let label = t(
+        'Send a %s notification',
+        serviceType !== NotificationActionService.SENTRY_NOTIFICATION
+          ? capitalize(serviceType)
+          : 'Sentry'
+      );
```

Comment:
nit: you could also make 3 blocks in here checking the 3 possible conditions. i do think the way i originally wrote it isn't that clear

```typescript
const getLabel = () => {
  switch (serviceType) {
    case NotificationActionService.SENTRY_NOTIFICATION:
      return t('Send a Sentry notification');
    case NotificationActionService.OPSGENIE:
      return t('Send an Opsgenie notification');
    default:
      return t('Send a %s notification', capitalize(serviceType));
  }
}
```